src/CodeGen_Hexagon.cpp

```diff
@@ -531,6 +534,8 @@ void CodeGen_Hexagon::init_module() {
 
         // Broadcasts
         { IPICK(is_128B, Intrinsic::hexagon_V6_lvsplatw), u32v1,  "splat.w", {u32} },
+        { IPICK(is_128B, Intrinsic::hexagon_V6_lvsplatb), u8v1,   "splat.b", {u8}  },   // v62 - Broadcasts
```

Comment:
Hmm, it's possible this won't work. What I expected to happen is that generating a call generates a call to that declaration, which will resolve to the non-weak function later. But, that seems suspicious, especially when inlining optimizations happen.

We need the target flag anyways for v60. Maybe define_hvx_intrinsic should check for an existing function and remove if it if it exists. Seems hacky though...